contracts/marketplace/lib.rs

```diff
@@ -268,20 +274,16 @@ pub mod marketplace {
 
         /// Update the length of the auction.
         ///
-        /// Can only be called by an account with an Admin role
-        /// and only if the contract is currently halted
+        /// Requires `Role::Admin`.
         #[ink(message)]
         pub fn set_auction_length(&mut self, new_auction_length: BlockNumber) -> Result<(), Error> {
-            if self.is_halted() {
```

Comment:
why do we allow for changing auction length without halting?

Reply:
both halting and the setter are guarded by the same Role:Admin, so I take it's not necessary, the same account will perform both actions if ever, and as it will likely be under an emergency scenario it's implicit the contracts will be halted